plugins/plugin-languageserver/che-plugin-languageserver-ide/src/main/java/org/eclipse/che/plugin/languageserver/ide/service/TextDocumentServiceClient.java

```diff
@@ -278,6 +281,20 @@ public Throwable getCause() {
     };
   }
 
+  public Promise<List<SnippetResult>> getSnippets(SnippetParameters params) {
+    return Promises.create(
+        (resolve, reject) -> {
+          requestTransmitter
+              .newRequest()
+              .endpointId(WS_AGENT_JSON_RPC_ENDPOINT_ID)
+              .methodName("textDocument/snippets")
```

Comment:
Please move to constants

Reply:
I don't think that makes sense here without changing all other mesage names, too.